wazo_call_logd/plugins/retention/listener.py

```diff
@@ -0,0 +1,15 @@
+from wazo_call_logd.database.queries.retention import RetentionDAO
+import logging
```

Comment:
Also, imports should be sorted this way: https://peps.python.org/pep-0008/#imports
In your case that would look like this:
```python3
import logging

from dataclasses import dataclass

from wazo_call_logd.database.queries.retention import RetentionDAO
```